SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -2315,6 +2315,12 @@
                 "perftest"
             ]
         },
+        "OpenEuler": {
+            "license": "[BSD-3 License](https://github.com/pytorch/pytorch/blob/master/LICENSE)",
```

Comment:
How do we know they release their specs under this license? I tried a brief search and could find [their repo for PyTorch](https://gitee.com/src-openeuler/pytorch), but no licensing info, so I got curious.

Reply:
We talked to CELA about not finding a license in openEuler's repo for this package and their advice is to take it under PyTorch license.